resolver/resolver.cc

```diff
@@ -3399,8 +3400,14 @@ class ResolveTypeMembersAndFieldsWalk {
             }
         }
 
-        // Compute the resultType of all classes.
-        if (hierarchyMayHaveChanged) {
+        // Recompute selectively when possible, otherwise compute everything.
+        if (symbolsToRecompute.has_value()) {
+            Timer timeit(gs.tracer(), "resolver.computeExternalType");
+            // Ensure all symbols have `externalType` computed.
+            for (auto symbol : *symbolsToRecompute) {
+                symbol.data(gs)->unsafeComputeExternalType(gs);
+            }
+        } else {
```

Comment:
Is this code relevant anymore?

If `finalizeSymbols` is always running before this code in `ResolveTypeMembersAndFieldsWalk`, and either runs over all symbols or over all `symbolsToRecompute`, what's left that doesn't have an external type computed here? I guess maybe some of the attached class type members could get invalidated by `resolveAttachedClass`?

On the slow path (not the focus of this PR), we're looping over the entire symbol table twice: once in finalizeSymbols and once in ResolveTypeMembersAndFieldsWalk, and it seems like we could reasonably only do that once.